src/views/layout.html

```diff
@@ -42,19 +42,17 @@
     useTudorCrown: true,
     containerClasses: "govuk-width-container"
   }) }}
-  <header class="govuk-header {{ phaseBannerClasses }}" role="banner" data-module="govuk-header" id="top"></header>
-    <div class="govuk-width-container">
-      {{ govukPhaseBanner({
-        tag: {
-        text: phaseTagText
-      },
-      html: phaseBannerHtml,
-      classes: phaseBannerClasses
-      }) }}
-    </div>
+ 
```

Comment:
Looks like adding the banner to `beforeContent` overrides the back link on our form pages, so in this case it would be easier to just add the banner to the header after all.

```{% block header %}
  {{ govukHeader({
    homepageUrl: "https://www.gov.uk",
    serviceName: SERVICE_NAME,
    serviceUrl: SERVICE_URL,
    useTudorCrown: true,
    containerClasses: "govuk-width-container"
  }) }}
  <div class="govuk-width-container">
    {{ govukPhaseBanner({
      tag: {
        text: phaseTagText
      },
      html: phaseBannerHtml,
      classes: phaseBannerClasses
    }) }}
  </div>
{% endblock %}
```

Please could you squash your commits into one commit after you make this change?